src/poliastro/twobody/mean_elements.py

```diff
@@ -23,6 +25,10 @@ def get_mean_elements(body, epoch=J2000):
     # So we are reverting the conversion in the last line
     # This way at least we avoid copy pasting the values
     try:
+        if body not in SOLAR_SYSTEM_BODIES:
```

Comment:
I just realized that `Sun` is an instance of `SolarSystemPlanet`... I wonder if we should change that somehow 😅

Reply:
In any case, what's the point of checking this list first? The line

```
body_index = PLAN94_BODY_NAME_TO_PLANET_INDEX[name]
```

tells me that we could replace this check by

```
if name not in PLAN94_BODY_NAME_...
```

and so we wouldn't need `SOLAR_SYSTEM_BODIES`. Right?

Reply:
I recall the reason was to handle if someone inputs `body=None` or any other invalid body such that

```python
name = body.name.lower()
```
would fail in the next line, since then we would get the error:

```bash
AttributeError: 'NoneType' object has no attribute 'name'
```

So would it be good to check for `body` first, instead of `name`?

Reply:
I think we shouldn't shield against every possible misuse of the API. The documentation and the signature are very clear:

https://github.com/poliastro/poliastro/blob/bebbf617e0ebfb797ba1c593b8241e4cf0f9c466/src/poliastro/twobody/mean_elements.py#L10-L16

If the user passes `None` as a `body`... `AttributeError: 'NoneType' object has no attribute 'name'` is a perfect explanation of what happened :)

Reply:
Thanks for this suggestion!

Now, it seems that there might be no need of checking `if name not in ...` since it's already inside a try-except block? Instead we could add to the existing error message stating that it must be an instance of `SolarSystemPlanet` and change `Sun` to not be an instance of `SolarSystemPlanet`?